src/components/pages/Bookings/__specs__/BookingsRecap.spec.jsx

```diff
@@ -11,6 +11,7 @@ import { configureTestStore } from 'store/testUtils'
 import { bookingRecapFactory, venueFactory } from 'utils/apiFactories'
 import { getNthCallNthArg } from 'utils/testHelpers'
 
+import NotificationContainer from '../../../layout/Notification/NotificationContainer'
```

Comment:
utiliser les import absolut
```suggestion
import NotificationContainer from 'layout/Notification/NotificationContainer'
```